### PR TITLE
Fix mixups in metrics documentation

### PR DESCRIPTION
**Pull request by tomMoulard**
### What does this PR do?

This PR corrects the documentation examples to remove superfluous quotes (`"`) in YAML.
It also fixes the order of some example tabs.

This PR also remove documentation for `Configuration reload failures` and `Last Configuration Reload Failure` metrics that are not used.

Finally, this PR adds documentation for the `TLS certificates expiration` metric that was missing.

### Motivation

Have proper documentation.

### More

- ~[ ] Added/updated tests~
- [X] Added/updated documentation

### CODE REVIEW

**Review by rtribotte — APPROVED** [review]
LGTM 👌

File: docs/content/observability/metrics/overview.md [hunk 1]
```diff
@@ -180,18 +181,120 @@ traefik_entrypoint_open_connections
...
+```
+
+```statsd tab="StatsD"
+# Default prefix: "traefik"
+{prefix}.router.request.tls.total
+```
+
+### Request Duration Histogram
+
+Request process time duration histogram on a router.
```
Comment (mpl) [inline docs/content/observability/metrics/overview.md:241]:
```suggestion
Request processing duration histogram on a router.
```

File: docs/content/observability/metrics/overview.md [hunk 2]
```diff
@@ -214,15 +317,30 @@ traefik_service_requests_total
...
 ```
 
+```statsd tab="StatsD"
+# Default prefix: "traefik"
+{prefix}.service.request.tls.total
+```
+
 ### Request Duration Histogram
+
 Request process time duration histogram on a service.
```
Comment (mpl) [inline docs/content/observability/metrics/overview.md:344]:
```suggestion
Request processing duration histogram on a service.
```

File: docs/content/observability/metrics/overview.md [hunk 3]
```diff
@@ -128,15 +113,30 @@ traefik_entrypoint_requests_total
...
 ```
 
+```statsd tab="StatsD"
+# Default prefix: "traefik"
+{prefix}.entrypoint.request.tls.total
+```
+
 ### Request Duration Histogram
+
 Request process time duration histogram on an entrypoint.
```
Comment (mpl) [inline docs/content/observability/metrics/overview.md:140]:
```suggestion
Request processing duration histogram on an entrypoint.
```

**Review by mpl — APPROVED** [review]

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

**Pull request merged**